authors/settings.py

```diff
@@ -189,3 +193,6 @@
 EMAIL_USE_TLS = True
 EMAIL_HOST_USER = config('EMAIL_HOST_USER')
 EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
+
+TWITTER_CONSUMER_KEY = config('TWITTER_CONSUMER_KEY')
```

Comment:
@tomuhenry Add these the `app.json`. Sync with @myrdstom to help with this and these will be added to the `heroku environment variables` and `.env.example` file.